apps/backoffice-v2/src/lib/blocks/hooks/useUbosBlock/useUbosBlock.tsx

```diff
@@ -1,30 +1,19 @@
 import { useMemo } from 'react';
 import { createBlocksTyped } from '@/lib/blocks/create-blocks-typed/create-blocks-typed';
+import { WarningFilledSvg } from '@/common/components/atoms/icons';
 
-export const useUbosBlock = ubos => {
-  return useMemo(() => {
-    if (!Array.isArray(ubos) || !ubos?.length) {
-      return [];
-    }
+type Ubo = {
+  name?: string;
+  percentage?: number;
+  type?: string;
+  level: number;
+};
 
-    return createBlocksTyped()
-      .addBlock()
-      .addCell({
-        type: 'block',
-        value: createBlocksTyped()
-          .addBlock()
-          .addCell({
-            type: 'heading',
-            value: 'UBOs',
-          })
-          .addCell({
-            type: 'subheading',
-            value: 'Registry-provided Data',
-            props: {
-              className: 'mb-4',
-            },
-          })
-          .addCell({
+export const useUbosBlock = (ubos?: Ubo[], message?: string) =>
```

Comment:
100% should do this for ubos, not certain about message. Ubos is not optional it is just something that can be undefined.
```suggestion
export const useUbosBlock = (ubos: Ubo[] | undefined, message: string | undefined) =>
```